tests/unit/config_test.py

```diff
@@ -191,6 +191,28 @@ def test_invalid_list_of_strings_format(self):
                 )
             )
 
+    def test_config_image_and_build_raise_validation_error(self):
```

Comment:
This is a duplicate test, we already have the image and build case covered here: https://github.com/mrfuxi/compose/blob/image-dockerfile-validation/tests/unit/config_test.py#L131